plugins/spu2-x/src/SndOut.cpp

```diff
@@ -390,8 +390,8 @@ void SndBuffer::Init()
 
     // clear buffers!
     // Fixes loopy sounds on emu resets.
-    memset(sndTempBuffer, 0, sizeof(StereoOut32) * SndOutPacketSize);
-    memset(sndTempBuffer16, 0, sizeof(StereoOut16) * SndOutPacketSize);
+    memset(static_cast<void *>(sndTempBuffer), 0, sizeof(StereoOut32) * SndOutPacketSize);
+    memset(static_cast<void *>(sndTempBuffer16), 0, sizeof(StereoOut16) * SndOutPacketSize);
```

Comment:
I'm not sure these lines are even needed. The constructors for StereoOut32 and StereoOut16 should already have zeroed things out.